content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,545 @@
+
+## Installation
+
+### Pre-requisites & Supported Topologies
+
+Ztunnel proxies are automatically installed when one of the supported installation methods is used to install Istio Ambient mesh.  The minimum Istio version required for the functionality described in this guide is 1.18.0. At this time, the ambient mode is only supported for deployment on Kubernetes clusters, support for deployment on non-Kubernetes endpoints such as Virtual machines is expected to be a future capability. Additionally, only single cluster deployments are supported for Ambient mode.  Some limited multi-cluster scenarios may work currently in ambient mode but the behavior is not guaranteed and official support for multi-cluster operation is a future capability. Finally note that Ztunnel based L4 networking is primnarily focused on East-West mesh networking and can work with all of Istio's North-South networking options including Istio-native ingress and egress gateways as well as Kubernetes native Gateway API implementation. 
+
+{{< tip >}}
+A single Istio mesh can include pods and endpoints some of which operate using the sidecar proxy mode while others use the node level proxy of the Ambient architecture. 
+{{< /tip >}}
+
+### Understanding the Ztunnel Default Configuration
+
+<< Consider breaking this out into bullets for easier reading TODO >>
+
+One of the goals for the ztunnel proxy design is to provide a usable configuration out of the box with a fixed feature set and that does not require much, or any, custom configuration. Hence currently there are no configuration options that need to be set other than the `ambient` profile setting. Once this profile is used, this in turn sets sets 2 internal configuration parameters (as illustrated in the examples below) within the istioOperator which eventually set the configuration of the `ambient` mesh. In future there may be some additional limited configurability for ztunnel proxies. For now, the pod to ztunnel proxy networking (sometimes also called ztunnel redirection), ztunnel proxy to ztunnel proxy networking as well as ztunnel to other sidecar proxy networing all use a fixed default configuration which is not customizable. In particular, currently, the only option for pod to ztunnel networking setup is currently via the `istio-cni` and only via an internal ipTables based ztunnel traffic redirect option. There is no option to use `init-containers` unlike with sidecar proxies. Alternate forms of ztunnel traffic redirect such as ebpf are also not currently supported, although may be supported in future. Of course, once the baseline `ambient` mesh is installed, features such as Authorization policy (both L4 and L7) as well as other istio functions such as PeerAuthentication options for mutual-TLS are fully configurable similar to standard Istio.  In future release versions, some limited configurability may also be added to the ztunnel proxy layer. 
```

Comment:
Once this profile is used, this in turn sets sets 2 internal configuration parameters (as illustrated in the examples below) within the istioOperator which eventually sets the configuration of the `ambient` mesh. In the future there may be some additional limited configurability for ztunnel proxies.

Reply:
this section has been removed from this guide. There will be a separate installation sub-guide which is being covered by a separate PR.